### ISSUE TITLE
Do sympy support polar plot?

### ISSUE THREAD

**Issue by Malkhan52** [sympy/sympy#14477]
like I want to plot r=sin(theta)+cos(theta) from 0 to pi

Also, I want to know how to use degree value of **pi**

**Comment by normalhuman** [comment]
SymPy does not have a polar plot function. See the [available plotting functions](http://docs.sympy.org/latest/modules/plotting.html). 

Any polar plot can be created as a parametric plot, using `(r*cos(theta), r*sin(theta))` for x and y coordinates. Example:
```
var('theta')
r = cos(theta) + sin(theta)
from sympy.plotting import plot_parametric
plot_parametric((r*cos(theta), r*sin(theta)), (theta, 0, pi))
```
To have degrees, you'd need to put `180/pi` in places. 

To get real polar plots, use [matplotlib](https://matplotlib.org/gallery/pie_and_polar_charts/polar_demo.html) directly. It provides polar grid, degree notation, etc. 

SymPy does not implement its own plotting, it only passes information to matplotlib. This is convenient but often restrictive. 

If the goal is to plot a function that comes from SymPy computations, use `lambdify` or `ufuncify` to creat a Python callable that will process the data. (`ufuncify` is more efficient with NumPy arrays, but supports fewer SymPy functions).

**Comment by nicoguaro** [comment]
I think that it might be possible to add the polar plot in [plot.py](https://github.com/sympy/sympy/blob/master/sympy/plotting/plot.py) in line 885, wouldn't it?

**Comment by jksuom** [comment]
I agree that it would be possible to add keyword `polar=True` but I'm not sure if that would suffice. Maybe something else should also be added or changed.

**Comment by Malkhan52** [comment]
Yes! @jksuom  I am planning same for degree too, like to enable values in degrees `degree=True`
I think it would be fine.
